index.js

```diff
@@ -51,7 +59,23 @@ function arg(opts, {argv, permissive = false} = {}) {
 			}
 
 			if (!(argName in handlers)) {
-				if (permissive) {
+				// Handle repeating boolean flags
+				const shortArg = argName.substring(0, 2);
+				const rest = argName.substring(2);
+				if (shortArg in handlers &&
+					Array.isArray(handlers[shortArg]) &&
+					rest.length > 0 &&
+					rest === argName.substring(1, 2).repeat(rest.length)
```

Comment:
It doesn't appear this PR is designed to support `-abc` right? Should it instead be iterating character by character until it sees a non-boolean, then treat the rest as the value of the first non-boolean key?

Reply:
Yeah, as of now `-abc` would be invalid because single-hyphen flags should be one character.

Expanding `-abc` to equal `-a -b -c` isn't a bad idea, but should be opened as another issue.